.github/workflows/rust.yml

```diff
@@ -61,11 +61,34 @@ jobs:
           command: test
           args: --all-features --no-fail-fast
 
-  test-integration-ignored:
+  # TODO(shonfeder): remove duplication once GitHub addresses
```

Comment:
IIUC, we could DRY this out *now* if we want to define our own action for this, but I'm not sure if we want to complicate the CI config that way. I'm up for it tho, if anyone thinks it's preferable.